static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -0,0 +1,136 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import Placeholder from 'sentry/components/placeholder';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjectFromSlug from 'sentry/utils/useProjectFromSlug';
+import {getGroupDetailsQueryData} from 'sentry/views/issueDetails/utils';
+
+import type {TimelineEvent} from './useTraceTimelineEvents';
+
+interface TraceIssueEventProps {
+  event: TimelineEvent;
+}
+
+export function TraceIssueEvent({event}: TraceIssueEventProps) {
+  const organization = useOrganization();
+  const project = useProjectFromSlug({organization, projectSlug: event['project.name']});
+  const issueId = event['issue.id'];
+  // We are view issue X and loading data for issue Y. This means that this call will not be cached,
+  // thus, it is a little bit slow to render the component.
+  // XXX: Create new endpoint that only fetches the metadata for the group
```

Comment:
I may be able to include the data within the `events` endpoint and save this call altogether.